### ISSUE TITLE
Autoloader Help

### ISSUE THREAD

**Issue by theskyfloor** [ARTACK/color#1]
Hey, perhaps I am missing something but when I try to initiate these classes and then use your example code I just get a 'class not found' error. Any advice would be appreciated!

**Comment by scuben** [comment]
Can you provide your code and the exact error reported by php? Also did you require the composers autoload.php file?

**Comment by theskyfloor** [comment]
I figured it out... sorry for bothering you! Do you have any recommendations for converting RGB to CMYK? It seems that no matter what library I use I cannot get output to screen match in Photoshop. Thanks for your speedy response!

**Comment by scuben** [comment]
From which RGB values are you converting to CMYK? What do you get with this library and what do you expect e.g. have in photoshop?

**Comment by scuben** [comment]
I just checked the RGB > CMYK conversion with the library and photoshop. Well you have absolutely right its not the same color. But all the online converters and formula I implemented seems to produce the slightly different green. As far as I know this is somewhat limited to green'ish colors. There are some color profile that help to get a more accurate conversion but this library does not use any of those.
